corehq/apps/linked_domain/README.md

```diff
@@ -1,9 +1,65 @@
 # Linking Domains
 
-* Ability to sync models between two domains (master domain -> linked domain)
-* Can be within the same HQ instance or between remote instances.
-
-# Remote linking
+Linked project spaces allow project spaces to share much of their data. You configure a controlling upstream domain
+(called "master" in most code, though we are replacing that over time) and one or more downstream domains
+(called "linked" in most code, similarly being replaced). You can then overwrite downstream content with upstream content.
+
+Linked domains have two major use cases:
+
+1. Setting up a practice domain and then a production domain
+1. Multi-geography projects that have one upstream domain and then a downstream domain for each of several different locations.
+
+Most domain linkages are "local",  with all domains residing in the same HQ cloud environment.
+There are also "remote" links, where the upstream domain is in a different environment than the downstream domains.
+Not all functionality is supported by remote linked domains, but most of it is. See below for more on remote
+linking.
+
+## User Interface
+
+The upstream domain looks like any other domain, but downstream domains have a more limited UI, because most of
+comes from their upstream domain and much of that shared content is read-only (see below for details on specific
+data models). Both upstream and downstream domains have a settings
+page (Project Settings > Linked Project Spaces) used to overwrite content.
+
+Overwriting can be triggered from either the upstream domain ("pushing" / "Enterprise Release Management") or any
+downstream domain ("pulling"). Individual downstream domains can be pushed/pulled, and individual data types can be
+pushed/pulled.
+
+To enable linked domains, turn on the feature flag
+[LINKED_DOMAINS](https://github.com/dimagi/commcare-hq/blob/966b62cc113b56af771906def76833446b4ba025/corehq/toggles.py#L1497).
+To link two domains, copy an application from the upstream domain to the desired downstream domain, checking the
+checkbox "Copy as linked application." This is a legacy workflow, leftover from when linked domains **only**
+supported applications. Remote domain linkages cannot be created via the UI; see below for details.
+
+## Data models
+
+Supported data types are defined in
+[LINKED_MODELS](https://github.com/dimagi/commcare-hq/blob/966b62cc113b56af771906def76833446b4ba025/corehq/apps/linked_domain/const.py#L13):
+
+- Applications
+- Reports
+- Lookup tables
+- User roles
+- Custom data fields for users, products, and locations
+- Feature flags and privileges
+- Case search settings
+
+Of these, apps and reports need to be linked individually, from the app settings and edit report UIs, and are
```

Comment:
I hadn't seen this before - is this a new thing for COVID?  Is it some option on the "edit reports" page that asks if you want to pull reports from upstream?

Reply:
Yeah, @proteusvacuum added this recently for COVID: https://github.com/dimagi/commcare-hq/pull/27569